config/samples/dataplane_v1beta1_openstackdataplanenodeset_customnetworks.yaml

```diff
@@ -24,17 +24,20 @@ spec:
       hostName: edpm-compute-0
       ansible:
         ansibleHost: 192.168.1.5
-        ansibleVars:
-          ctlplane_ip: 192.168.1.5
-          internalapi_ip: 172.17.0.101
-          storage_ip: 172.18.0.101
-          tenant_ip: 172.19.0.101
-          external_ip: 172.20.12.76
-          fqdn_internalapi: edpm-compute-1.example.com
-          ansible_ssh_transfer_method: scp
   networkAttachments:
     - ctlplane
   nodeTemplate:
+    networks:
```

Comment:
This also has to go to `nodes` section. Also it seems `preProvisioned: true` is missing in this one.

Reply:
As long as I remove the `fixedIP` from the `nodeTemplate` section, this would still be valid though right?

And I added `preProvisioned: true` in the latest iteration as well.

Reply:
No it won't work as with `preProvisioned: true` the IPAM provided `fixedIP` can be different from what you use in `ansibleHost` and aee can't reach edpm nodes using `ansibleHost`. For `preProvisoned: true` you can't have networks in `nodeTemplate` section.